src/controller/WriteInteraction.h

```diff
@@ -99,19 +99,26 @@ CHIP_ERROR WriteAttribute(SessionHandle sessionHandle, chip::EndpointId endpoint
     ReturnErrorOnFailure(app::InteractionModelEngine::GetInstance()->NewWriteClient(handle, callback.get()));
     if (sessionHandle.IsGroupSession())
     {
-        // TODO : Issue #11604
-        return CHIP_ERROR_NOT_IMPLEMENTED;
-        // ReturnErrorOnFailure(
-        // handle.EncodeAttributeWritePayload(chip::app::AttributePathParams(clusterId, attributeId), requestData));
+        ReturnErrorOnFailure(
+            handle.EncodeAttributeWritePayload(chip::app::AttributePathParams(clusterId, attributeId), requestData));
     }
     else
     {
         ReturnErrorOnFailure(
             handle.EncodeAttributeWritePayload(chip::app::AttributePathParams(endpointId, clusterId, attributeId), requestData));
     }
+
     ReturnErrorOnFailure(handle.SendWriteRequest(sessionHandle));
 
     callback.release();
+
+    if (sessionHandle.IsGroupSession())
+    {
+        // Manually call success callback since OnReponse won't be called in WriteClient for group
+        app::ConcreteAttributePath aPath;
+        onSuccessCb(aPath);
```

Comment:
This deserves a TODO comment: we can't require that onSuccessCb be handed a ConcreteAttributePath if we are allowing group writes; it needs to get something else.....

But maybe the real answer is that we should _not_ in fact be calling `onSuccessCb` for groups?  As in, the only callbacks a group caller can expect is OnError if there's a failure before send and an OnDone when send is done.

Reply:
Yeah you're right on this. However with the current Test framwork, we expect the `stopWaiting` call to be issued which is only done in the `OnSuccessCb. If Not triggered manually the test/write interaction will fail with a Timeout error. Issue #11975.